Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/TaskState.kt

```diff
@@ -0,0 +1,47 @@
+package de.rki.coronawarnapp.task
+
+import org.joda.time.Instant
+import kotlin.reflect.KClass
+
+interface TaskState {
+    val request: TaskRequest
+    val createdAt: Instant
+    val startedAt: Instant?
+    val completedAt: Instant?
+    val error: Throwable?
+    val result: Task.Result?
+    val executionState: ExecutionState
+    val type: KClass<out Task<*, *>>
+
+    val isFinished: Boolean
+        get() = executionState == ExecutionState.FINISHED
```

Comment:
I see `completedAt` but `isFinished`. How about aligning this? So either, `finishedAt` (which I would prefer; `completedAt` has a notion of "success") or `isCompleted`.